cmake/FindPackageMessage.cmake

```diff
@@ -0,0 +1,48 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
Why did you add this, it is part of your `CMake` installation....

Reply:
Good question, I got an error about it, so cmake didn't find it with the current way it is executed in the freerdp build script. Let me look into what is wrong.

Reply:
Fixed.